src/io/flutter/FlutterErrorReportSubmitter.java

```diff
@@ -96,7 +96,7 @@ public boolean submit(@NotNull IdeaLoggingEvent[] events,
     final DataContext dataContext = DataManager.getInstance().getDataContext(parentComponent);
     final Project project = PROJECT.getData(dataContext);
     if (project == null) {
-      fail(((Consumer<SubmittedReportInfo>)consumer));
```

Comment:
This is required to compile with current AS sources.

Reply:
Reverted (and the two places below as well).